source/_integrations/decora_wifi.markdown

```diff
@@ -17,24 +17,16 @@ Supported devices (tested):
 - [DW6HD1-BZ](https://www.leviton.com/en/products/dw6hd-1bz) (Decora Smart Wi-Fi 600W Dimmer)
 - [DW15S-1BZ](https://www.leviton.com/en/products/dw15s-1bz) (Decora Smart Wi-Fi 15A Switch)
 - [DW15P-1BW](https://www.leviton.com/en/products/dw15p-1bw) (Decora Smart Wi-Fi Plug-in Outlet)
+- [DW4SF-1BW](https://www.leviton.com/en/products/dw4sf-1bw) (Decora Smart Wi-Fi Fan Speed Controller)
 
-To enable these lights, add the following lines to your `configuration.yaml` file:
-
-```yaml
-# Example configuration.yaml entry
-light:
-  - platform: decora_wifi
-    username: YOUR_USERNAME
-    password: YOUR_PASSWORD
-```
-
-{% configuration %}
-username:
-  description: Your "My Leviton" app email address/user name.
-  required: true
-  type: string
-password:
-  description: Your "My Leviton" app password.
-  required: true
-  type: string
-{% endconfiguration %}
+## Setup
+
+You will need your MyLeviton login information (username, which is usually your email address, and password) to use this module.
```

Comment:
```suggestion
You will need your MyLeviton login information (username, which is usually your email address, and password) to use this integration.
```

Reply:
I wonder, is this set/prerequisites section really needed?

Reply:
Possibly not. I was following the format used by the doc page for Blink; it looks like other integrations don't put a heading over this portion of the doc.